components/checkbox/stories/checkbox.stories.js

```diff
@@ -84,7 +105,34 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
+const CheckboxGroup = ({
```

Comment:
I think this works in kitchen sink style too. Do you think we should include a disabled example? I could go either way...might be nice to have the coverage though.